cmd/sql/flow.go

```diff
@@ -74,6 +75,40 @@ func buildFlagsAndMountDirs(projectDir string, setProjectDir, setAirflowHome, se
 		flags["project-dir"] = projectDir
 	}
 
+	if mountGlobalDirs {
+		configCommandString := []string{"flow", "config"}
+		configFlags := make(map[string]string)
+		configFlags["project-dir"] = projectDir
+
+		args := []string{"airflow_home"}
+		exitCode, output, err := sql.CommonDockerUtil(configCommandString, args, configFlags, mountDirs, true)
+		if err != nil {
+			return nil, nil, fmt.Errorf("error running %v: %w", configCommandString, err)
+		}
+		if exitCode != 0 {
+			return nil, nil, sql.DockerNonZeroExitCodeError(exitCode)
+		}
+		airflowHomeAbs, err := sql.ConvertReadCloserToString(output)
+		if err != nil {
+			return nil, nil, err
+		}
+		mountDirs = append(mountDirs, strings.TrimSpace(airflowHomeAbs))
+
+		args = []string{"airflow_dags_folder"}
+		exitCode, output, err = sql.CommonDockerUtil([]string{"flow", "config"}, args, configFlags, mountDirs, true)
+		if err != nil {
+			return nil, nil, fmt.Errorf("error running %v: %w", configCommandString, err)
+		}
+		if exitCode != 0 {
+			return nil, nil, sql.DockerNonZeroExitCodeError(exitCode)
+		}
+		airflowDagsFolderAbs, err := sql.ConvertReadCloserToString(output)
```

Comment:
@pankajkoti I noticed a couple of parts o the code we're not covering with tests. Would it make sense to extract these into smaller functions and increase the test coverage? Do you have any other ideas?

Reply:
yes, good idea. Refactored and we have a good coverage now.